### PR TITLE
sqs queue to terraform

### PR DESCRIPTION
**Pull request by AmyCGT**
converted dlp-training-queue and dead-letter-queue to terraform

### CODE REVIEW

**Review by farisdurrani — APPROVED** [review]

**Comment by farisdurrani** [comment]
All good for me. Just update the branch and I recommend waiting for Karthik's approval

File: dlp-terraform/sqs/sqs.tf [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  }
+
+  required_version = ">= 1.2.0"
+}
+
+provider "aws" {
+  region = "us-west-2"
+}
+
+resource "aws_sqs_queue" "dlp_training_queue" {
```
Comment (karkir0003) [inline dlp-terraform/sqs/sqs.tf:16]: @thomaschin35 @AmyCGT this shouldn't be there

**Review by karkir0003 — CHANGES_REQUESTED** [review]
remove extra sqs resource

File: .gitignore [hunk 2]
```diff
@@ -194,7 +194,6 @@ frontend/playground-frontend/src/backend_outputs/*
 
 # Terraform Files
 dlp-terraform/.terraform
-dlp-terraform/outputs
```
Comment (karkir0003) [inline .gitignore:197]: @thomaschin35 you're sure this is correct

**Review by karkir0003 — COMMENTED** [review]
answer this quick question of mine and then I'll approve the PR

File: .gitignore (hunk 2 above)
Comment (thomaschin35) [reply .gitignore:197]: Yes this is correct

**Review by karkir0003 — APPROVED** [review]

**Pull request merged**